src/main/java/seedu/address/logic/commands/Command.java

```diff
@@ -43,4 +43,16 @@ public void setData(Model model) {
     protected void indicateAttemptToExecuteIncorrectCommand() {
         EventsCenter.getInstance().post(new IncorrectCommandAttemptedEvent(this));
     }
+
+    /**
+     * Does this command modify data in TaskBook? true or false
+     * @return
+     */
+
+
+    /***
+     * To return the command word child classes of Command will have
+     * @return
+     */
+    public abstract String getCommandWord();
```

Comment:
Okay, I do not mind that much if `getCommandWord()` is part of the interface of `Command`. (even though knowing the command word is more the responsibility of the *Parser*)

However, given that this is user-interface specific only, must we really make this `abstract`? It seems like we can provide a decent default implementation with `getClass().getName()` or `this.toString()` or something.